docs/roadmap.md

```diff
@@ -0,0 +1,18 @@
+## Next Week
+
+- Allow table name to be customizable.
+- Incremental CDC-style backfill to avoid re-pulling full historical datasets.
+- Use JSONB for storing all of the data and allow user to create generated columns as necessary for future proofing. This ensures that read-only columns stay read-only because they are generated and also allow user to drop them without breaking the sync.
```

Comment:
Duplicate all the data for each given row in a read-only JSONB column that only the library can modify?